utils/geometry.py

```diff
@@ -112,7 +112,7 @@ def intersects(self,center,radius):
 			return False
 	# self is the center of circle, checks whether the line made by the point1 and point2 intersects the circle
 	def intersects(self,point1,point2,radius):
-		# Source of algorithm used: http://stackoverflow.com/questions/1073336/circle-line-collision-detection 
+		# Source of algorithm used: http://stackoverflow.com/questions/10733336/circle-line-collision-detection 
```

Comment:
@RKJenamani why change this link?

Reply:
Someone must have done it by mistake. Did not notice.

Reply:
change this to previous one